docs/network-spec/figure/ConnectionManagerInteractions.drawio

```diff
@@ -0,0 +1 @@
s1nflOZ/ieM07+P80TV94DwfbNJa7n3VQ8vItu94iHhYNf+0aisOHZ36D/OilevSFJiHrAJrwxk0aJEKhrklvYPdXfCO8zWQL9DlMv5WPwo7+Erdmn8unyg7EQ+WdmPVcvS2wjo63yYTWaQRzIAqSOU0PepduXQmNgjT8KT+JyXD4pV/ikD1jaZW2x2GVoypyFePLX4FfVtrfbZIEL5XT1tkJm5obIQKkG0EZY9iHXGRp3cVbnmDwjoZJ9xzdTY7wKRizoCMZbxCF81UGNUy1mQHeZSAQMlC/5V8sw+k09xO6Cf8Oxjt5mZXwHmHC8d0NfpBtD3p1qMIjEBdW4n7VwmocfC8GAQsgH0pqGHFIONGOfGh5UNGvbQHbr/zJMuPZbEPTGxXSulA60JSuaVsGtdPix4xM6MQYP8YEOxh0FD88V3YgLKyiGk8cQzzBqkt3Fk+QHvLlKG6I/FnEZ2c80ohZeBivKmfrIZ91VyrrQu7jVbyiikJ4U3O/NSlfNo+q/mFP2iyOq9q0bV2b/ZEDfGa3mVHX7DZTzx+DrtzGduSOLuJQpaNFLJI6moC+elqoVY9KuT98DlYsF0iuxBuSOA12D14lR93qzLUb6gz2pjMfaOo4B3UW7LTKTS1PUFUzPa3wbWjLdNvzWvPt2ywDzkwoCjabcCKaP4ZRaYwyLaerqbiImyFr4ecDk983o+2whOBuKDgnJv1T8NMMEGqgcTuZ0PUrZbI+MHd2wWQx8L3TqGvBgC2CzsZN/fMGWRcE/k5DmrOhB0Y5BJ5LYTn764nlmIKvkbOi3oAcIk0Dv6/GWZdnNSgWzOJJHEkU9q1GYQ+RLNRU9WdDQyD2strJ+Xgq1NnTI01+vmW+5PqymzkmNQGDm/WmJKxHK009ooo7i+gzJwpVzqAQDb15r14Zit0BHzetAAq6NAIWAL5MmKCPDjEmTtEEXeNEyyWHOZrKs7Iz1DahQRO4f/z44LvuEWxIt6CKgZhK0aLt1CpjUasQBmq7litL2VNnNFQsHWgBT6sr7ZecEztNclch3dNrjKLssVkHK8nm3R/beFcPCSbf5zuYGE1yfd5mvTwf/4Iww1KQj4gwaC0+Q4B+La8X1ZR4m4rQNd2umR+VlGuXSuaPwV4rf5L8oqEJY20JpTVhxPUYCazMNiRT6aj0OYK+hZEi2cK+9KeYZH8EEztG9Z4pTx3tKswSEtqoFgg7Hcs5kHCa8LVhblsEieIgfzv+tsdm1E0Bu8sRJGdPRnWq+Tu2PIKEbWz5++v+jXHe8W1FbjNc3z8y1Tny66mEgNzMNI3In30x2uyMNoN8CNfPOZ4ryJ7QH1u6Sf/DAf7eBOr5TV41qDsHQB34wv5PNGNsESU6iDzjDJiNh8RsubYovhkerafBZrF7P/FIGl3eg8nO1SCy3xciK1l7wzF9gyQP1EsaHIOxOzTfAdLEEQs4/sW4UCeEh72hMopwMCV+cw6G2joYBqBe0uAO5ukV6yldximd/jdmVjv9c3fwSumHe4B+QILk8kM3KSXEooZRkBFknW2MAntDQOq5cVGu+JXpKqlnQF1BqjIPEkL3akC1pzzSxjIUYoLbshY1c1QlDQ+q+hjGdpXQebhhujXlfK8SXb0D6Io8QrrBU2LJMF2MAp4BTcnQBBXKCOfUzx650BRPH5pB8HoSvTOV3lx1PK81ZGqSBodM7A/pR3KRBOfzqq7QhxTaYZep3bsPccv3O/MhVdLRPrTvRtj8yL0upRCVn4vwP1I/un6h/qeUKSF89z25juLh9r7n1EvqzvfMj9yv7wmNXoDvue61xj6FP2JI3v1P9hoCu4p9mqS+Yp+4Ub/+hy7H/w5M/h/O/aqu5smu5jrXM2RwplBHSFehTpM0eKrmDrPM3OguxH4FVJHPJn33n9Lq/c6ooiapL6ron4UqDjr/T/a9yxkIP4ooQsX7CG61au6VeZ8FKn+ObNkQqMt1WnujLmr4cHZB07P8i6V/R4Qz332v3NtOvUO1n7KlZGK6qOEdatA5W7JDIfsqZzzqOZaPWu2lNIhT9TQty1aXwbldzSDQJA3uQ4K01i2w0+2tbsncUYZsBvaDFmiu5tc6YJcG2nSl/z4DPdNqOV+ZCeO0nl7oq9MLvWaxoLN5Lg32lDEu+TzdUKFUlAKyoULcCizNaz3d+hVOJiAfxKjxoEbtKFsBOMBpC862LMltmMB3ZtSuZtThahJtp5RvJPDaJ255eyypsjeHmPUk5luhjowIM54jSz7fTC7Pu0wwA6jVJA4zmAHfeUezBgvaoYxByCfXimb6Hn7rJOYT/L8GyfI1T/D36qegjoBlYyhPxe8GyWyWwckp/8i1iCKmPygjx2UP3UOZwB6WxyIJf2xwPJuSd/BQV2h7g+fFV4JpjrJDr9e+qDswpglHMk6vfyMkjdRvoDbKkiBhmQWX6sSQoK3th+Fb6GyrqcVmbnXYdmBTtWCzzn8AYBY+ZwW55uo6YtMmpOTmpj2biMHdoTre39mmTURflVIXFTqrKcGa8mzDrS4V0D9YC+0Q3oW9HYT3/MTB4F3Jv227Lbqr1Lfh/JOu/Bvq6TfDcmZf2ZX3i4DhZaSZ7cVtBbdbvNLDVnAQKMM/DtFgxTNYWW87wbkNsuY+d4JrU9bOaCSREQU6Bxdo7uW0mBzIqZtuBne1+79BZWQGup68R1DbWiBEvoVhRRA56j49Q5U76Nq/oyLpYOOc7W33ZKO0lV+sgH6xCdDxhqhauEFYVxPLIN774Puf7+A1Pc1Ha7DhcU+VBnQM0RsOGolcX2o9LxiiekF91xH1ktLjgvXy9O1yMTUYAT3Dc89JxTy9LPKZsvDyrqKiUDWwhohpL9lzZeCXD5WQ8XBP9qnWq3UNwnCzST6dFaqOmxxxjnJLu2rLe7Hl1RZbiF5sGdpGm/76jbqG5t1KDyctrRfVQOUXm7QfYurbTvVR7E90TldvmNroKumL27DD8reHc42WP+Fsf/gH</diagram></mxfile>
```

Comment:
what is a `.drawio` file?

Reply:
Is the file responsible for generating the diagram I made. I included it here so we can import the file directly in draw.io and make the necessary changes without having to redo the diagram all over again.